lib/cocoapods/generator/embed_frameworks_script.rb

```diff
@@ -83,7 +83,7 @@ def script
 
   if [ -L "${source}" ]; then
     echo "Symlinked..."
-    source="$(readlink "${source}")"
+    source="$(readlink -f "${source}")"
```

Comment:
This solution is working. I used this workaround for a Flutter project running on 3.7.7.
Thanks, @chrisvasselli

Reply:
good job bro. Thanks!

Reply:
Hi, I cant fix it with my Flutter project. Could you help me please?

Reply:
Hey nickjun03,

> Hi, I cant fix it with my Flutter project. Could you help me please?

I managed to solve the issue by doing the following:

1. Download the cocoapods repository and extract it in a folder
2. make the change in embed_frameworks_script.rb as shown above
3. Follow the steps for 'Set up a local dev copy of CocoaPods' here : https://guides.cocoapods.org/using/unreleased-features . The real world walk through is very helpful. I did not have to perform 'git checkout swift' by the way. If you are missing bundle, you can install it by: `sudo gem install bundler:2.3.26`
4. In your Flutter's ios folder, you can perform `pod deintegrate` , delete podfile.lock, then do pod install but with the local installation, like this example: `/Users/orta/spiel/ruby/CocoaPods/bin/pod install`
5. What i also had to do for the build to be successful is, I had to open Pods.xcodeproj, and go to 'ObjectMapper' in 'Targets', and set iOS Deployment Target to be 11.0 . 

I hope this helps!

Reply:
Hello @chamodanethra thanks

My issue has been fixed

Reply:
This solution is working. You saved my life. Thanks!

Reply:
Flutter 3.7.10 has fixed this: https://github.com/flutter/flutter/issues/123890

Reply:
upgrade Flutter 3.7.10 solution for my

Reply:
Doesn't work in some cases: 
https://github.com/flutter/flutter/issues/124081#issuecomment-1515864512

Reply:
this works after spending days Thanks !!!! for flutter edit find file called 
Pods-Runner-frameworks.sh
and then search for this 

source="$(readlink "${source}")"

and add the -f 

Enjoy <3

Reply:
Thank you soo much. Works like a charm.